lib/src/components/supa_socials_auth.dart

```diff
@@ -153,62 +153,43 @@ class _SupaSocialsAuthState extends State<SupaSocialsAuth> {
         providers.length,
         (index) {
           final socialProvider = providers[index];
+
+          Widget iconWidget = SizedBox(
+            height: 48,
+            width: 48,
+            child: Icon(socialProvider._iconData),
+          );
+
+          Color? foregroundColor = coloredBg ? Colors.white : null;
+          Color? backgroundColor =
+              coloredBg ? socialProvider._btnBgColor : null;
+          Color? overlayColor = coloredBg ? Colors.white10 : null;
+
+          Color? iconColor = coloredBg ? Colors.white : null;
+
           if (socialProvider == SocialProviders.google && coloredBg) {
-            return Padding(
-              padding: const EdgeInsets.symmetric(vertical: 8.0),
-              child: ElevatedButton.icon(
-                icon: Image.asset(
-                  'assets/logos/google_light.png',
-                  package: 'supabase_auth_ui',
-                  width: 48,
-                  height: 48,
-                ),
-                style: ButtonStyle(
-                  foregroundColor: MaterialStateProperty.all(Colors.black),
-                  backgroundColor: MaterialStateProperty.all(Colors.white),
-                ),
-                onPressed: () async {
-                  try {
-                    await supaClient.auth.signInWithOAuth(
-                      socialProvider.provider,
-                      redirectTo: widget.redirectUrl,
-                    );
-                  } on AuthException catch (error) {
-                    if (widget.onError == null) {
-                      context.showErrorSnackBar(error.message);
-                    } else {
-                      widget.onError?.call(error);
-                    }
-                  } catch (error) {
-                    if (widget.onError == null) {
-                      context.showErrorSnackBar(
-                          'Unexpected error has occurred: $error');
-                    } else {
-                      widget.onError?.call(error);
-                    }
-                  }
-                },
-                label: Text('Continue with ${socialProvider.capitalizedName}'),
-              ),
+            iconWidget = Image.asset(
+              'assets/logos/google_light.png',
+              package: 'supabase_auth_ui',
+              width: 48,
+              height: 48,
             );
+
+            foregroundColor = Colors.black;
+            backgroundColor = Colors.white;
+            overlayColor = Colors.white;
```

Comment:
Only for Google login button use a google logo image for icon and black for text. Specifying overlayColor prevents the background color of the image from showing up like this:

![image](https://user-images.githubusercontent.com/18113850/221524153-ab48ab9d-46de-41b3-98a2-7a91c07bbf19.png)